### ISSUE TITLE
Need Help

### ISSUE THREAD

**Issue by megamuzic** [squallooo/MT5#11]
I like your player very much. Is it possible to install it on sharing hosting?

**Comment by micbuffa** [comment]
Yes, you need to find a hosting service that accepts nodeJS applications.
There are a lot, but beware, multi track songs may take a lot of hard disk
space. Look at : https://github.com/joyent/node/wiki/node-hosting

2015-03-18 19:19 GMT+01:00 megamuzic <notifications@github.com>:

**Comment by mrjcleaver** [comment]
If https://github.com/squallooo/MT5/issues/5 was done in such a way that you streamed from Soundcloud rather than synced a copy of the files then no disk space would be needed. 

Not sure how feasible that is.

**Comment by micbuffa** [comment]
Indeed, I'll look at soundcloud support the next week, I'm a bit overloaded
by my work right now.

2015-03-19 17:04 GMT+01:00 Martin Cleaver <notifications@github.com>:

**Comment by megamuzic** [comment]
Hey man, im trying to deploy your app on microsoft azure and i get this error
 deployment failedfatal: cannot create directory at '.c9/metadata/preview-https:': Invalid argument

Could help?
 



     On Thursday, March 19, 2015 6:06 PM, micbuffa <notifications@github.com> wrote:
   

 Indeed, I'll look at soundcloud support the next week, I'm a bit overloaded
by my work right now.

2015-03-19 17:04 GMT+01:00 Martin Cleaver <notifications@github.com>:

**Comment by RobinVivant** [comment]
You can safely remove the .c9 directory, it contains Cloud9 IDE related
files.

**Comment by megamuzic** [comment]
Thank you for your help. 
Is it possible to read folder from distance server?I've changed multitrack to actual path in server.js and track.js but it doesn't work.
 


     On Saturday, March 21, 2015 4:02 AM, Robin Vivant <notifications@github.com> wrote:
   

 You can safely remove the .c9 directory, it contains Cloud9 IDE related
files.

**Comment by RobinVivant** [comment]
Nope, it handles only relative local folder paths.

**Comment by micbuffa** [comment]
I planned to add support for soundcloud files, I will give it a look this week I hpe.
